generators/app/templates/src/app/components/RadioGroup/components/RadioOption/interfaces.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export interface IRadioOption {
+  checked?: boolean;
+  children: React.FunctionComponent;
+  className?: string;
+  disabled?: boolean;
+  key: string | number;
+  inputClassName?: string;
+  labelClassName?: string;
+  label?: string | undefined;
+  name?: string;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  value: string;
+  [rest: string]: any; 
```

Comment:
Do you need this attribute?

Reply:
we can remove this, the interface doesn't limit you to add only the stuff you define.